### PR TITLE
Added html and css files for loading page

### PR DESCRIPTION
**Pull request by aastha271100**
**ISSUE NO** #2

### CODE REVIEW

File: Aastha Bansal_loadingpage/loadingpage.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      <div id="pan"></div>
+      <div id="handle"></div>
+    </div>
+    <div id="pancake">
+      <div id="pastry"></div>
+    </div>
+  </div>
+</div>
+</body>
+</html>
```
Comment (raksha009) [inline Aastha Bansal_loadingpage/loadingpage.html]: Plz add a new line after line 28

**Comment by raksha009** [comment]
Plz also add a video of the code running on yr system.

**Comment by aastha271100** [comment]
I have made the changes. Please check.

**Review by raksha009 — CHANGES_REQUESTED** [review]
Hey! The video has to bee added in the pr and not inside the folder.

**Review by raksha009 — CHANGES_REQUESTED** [review]
Dear, plz have a look at this pr. https://github.com/dscigdtuw/Indulge/pull/24

**Comment by aastha271100** [comment]
https://drive.google.com/file/d/1ldd38gxDR_CW_2ad94iMmsAOnhUD1q1E/view?usp=sharing

**Review by raksha009 — APPROVED** [review]
Great Work!

**Pull request merged**